### ISSUE TITLE
lnd fails to start

### ISSUE THREAD

**Issue by martinatime** [lightningnetwork/lnd#2980]
### Background

I have had a working lnd node running on a RaspiberryPi for almost a year now. It was working fine a few days ago but something happened and now lnd won't start. I upgraded the lnd version from 5.2 to 6 to see if that might resolve it but the issue still exists. Thank you for any assistance.

### Your environment
lnd version 0.6.0-beta commit=v0.6-beta
Linux RaspiNode 4.14.79-v7+ #1159 SMP Sun Nov 4 17:50:20 GMT 2018 armv7l GNU/Linux
Bitcoin Core Daemon version v0.17.1

### Steps to reproduce
Run "lnd"
```
Attempting automatic RPC configuration to bitcoind
Automatically obtained bitcoind's RPC credentials
2019-04-19 12:22:51.955 [INF] LTND: Version: 0.6.0-beta commit=v0.6-beta, build=production, logging=default
2019-04-19 12:22:51.955 [INF] LTND: Active chain: Bitcoin (network=mainnet)
2019-04-19 12:22:51.956 [INF] LTND: Shutdown complete
panic: invalid freelist page: 13364, page type is leaf

goroutine 1 [running]:
github.com/coreos/bbolt.(*freelist).read(0x3210700, 0x5a01b000)
        /Users/roasbeef/gocode/pkg/mod/github.com/coreos/bbolt@v1.3.2/freelist.go:266 +0x23c
github.com/coreos/bbolt.(*DB).loadFreelist.func1()
        /Users/roasbeef/gocode/pkg/mod/github.com/coreos/bbolt@v1.3.2/db.go:310 +0xf0
sync.(*Once).Do(0x318c480, 0x3075d28)
        /usr/local/go/src/sync/once.go:44 +0xb8
github.com/coreos/bbolt.(*DB).loadFreelist(0x318c3c0)
        /Users/roasbeef/gocode/pkg/mod/github.com/coreos/bbolt@v1.3.2/db.go:303 +0x48
github.com/coreos/bbolt.Open(0x31044b0, 0x30, 0x180, 0x15d7fc8, 0x30, 0x25, 0x31fe380)
        /Users/roasbeef/gocode/pkg/mod/github.com/coreos/bbolt@v1.3.2/db.go:280 +0x2ac
github.com/lightningnetwork/lnd/channeldb.Open(0x3104480, 0x25, 0x3075f24, 0x2, 0x2, 0x2, 0x0, 0x3100580)
        /Users/roasbeef/gocode/src/github.com/lightningnetwork/lnd/channeldb/db.go:131 +0x104
main.lndMain(0x0, 0x0)
        /Users/roasbeef/gocode/src/github.com/lightningnetwork/lnd/lnd.go:164 +0x3a8
main.main()
        /Users/roasbeef/gocode/src/github.com/lightningnetwork/lnd/lnd.go:477 +0x14
```

### Expected behaviour

lnd should have started

### Actual behaviour
It fails to start with the above stack trace.

**Comment by wpaulino** [comment]
Looks like you've encountered disk corruption. Since this happened while you were on v0.5.2, there's no way to recover your off-chain funds. The best you can do is recover you on-chain funds following [this guide](https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md#on-chain-recovery).

Once you start the new node on v0.6, you'll have access to SCBs (you can read more about them [here](https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md#off-chain-recovery)), which will allow you to recover your off-chain funds if this arises again.

**Comment by martinatime** [comment]
Thanks for the info. I followed the steps and I believe that I need to run the dropwtxmgr but no matter what I tried I couldn't figure out how to get it to run. It just doesn't seem to "install" based on the instructions in the guide. I installed Go. I set a GOPATH. Running the "go get" doesn't seem to install the dropwtxmgr. I have a feeling that something is assumed in those instructions. Thanks for any help on this.

**Comment by wpaulino** [comment]
> Running the "go get" doesn't seem to install the dropwtxmgr.

What's the output of the command? It'll install it into `$GOPATH/bin/`.

**Comment by martinatime** [comment]
admin@RaspiNode:~ $ echo $GOPATH
/home/admin/gocode
admin@RaspiNode:~ $ go version
go version go1.7.4 linux/arm
admin@RaspiNode:~ $ go get -v -u github.com/btcsuite/btcwallet/cmd/dropwtxmgr
github.com/btcsuite/btcwallet (download)
github.com/btcsuite/btcutil (download)
github.com/btcsuite/btcd (download)
Fetching https://golang.org/x/crypto/ripemd160?go-get=1
Parsing meta tags from https://golang.org/x/crypto/ripemd160?go-get=1 (status code 200)
get "golang.org/x/crypto/ripemd160": found meta tag main.metaImport{Prefix:"golang.org/x/crypto", VCS:"git", RepoRoot:"https://go.googlesource.com/crypto"} at https://golang.org/x/crypto/ripemd160?go-get=1
get "golang.org/x/crypto/ripemd160": verifying non-authoritative meta tag
Fetching https://golang.org/x/crypto?go-get=1
Parsing meta tags from https://golang.org/x/crypto?go-get=1 (status code 200)
golang.org/x/crypto (download)
package math/bits: unrecognized import path "math/bits" (import path does not begin with hostname)
github.com/btcsuite/btclog (download)
github.com/btcsuite/golangcrypto (download)
github.com/coreos/bbolt (download)
github.com/jessevdk/go-flags (download)

**Comment by martinatime** [comment]
it didn't install anything in the $GOPATH/bin/ directory sadly

**Comment by wpaulino** [comment]
Try these alternative steps to install the binary:
```bash
$ git clone https://github.com/btcsuite/btcwallet
$ cd btcwallet
$ GO111MODULE=on go install ./cmd/dropwtxmgr
```

**Comment by martinatime** [comment]
@wpaulino thanks for your help on this. I figured out my issue with go as I was using version 1.7 (as that is what apt-get installs) but I needed at least 1.9. I have installed 1.12+ and have been able to follow the droptxmgr steps. It reverted everything back to my wallet's origin which is just over a year old. It then ran through all of the old blocks but didn't find anything. Is there anything else I can try? Is there something I need to do after the "Forced In-Place Rescan" finished?
